include/cantera/kinetics/Arrhenius.h

```diff
@@ -370,14 +390,34 @@ class BlowersMaselRate final : public ArrheniusBase
 
     virtual void getParameters(AnyMap& node) const;
 
-    double evalFromStruct(const BlowersMaselData& shared_data) {
-        double deltaH_R;
+    void check(const std::string& equation, const AnyMap& node) override {
+        checkRate(equation, node);
+    }
```

Comment:
Sorry I missed this before, but there are several other functions in `BlowersMaselRate` that now need to be marked `override` (at least, to avoid getting warnings from Clang). Namely, `newMultiRate`, `type`, `setParameters`, and `getParameters`.

Reply:
No problem - fixed. Should probably switch to clang!